AIMSICD/src/main/res/values-en-rUS/translatable_strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="cell_info_title">Neighboring Cells</string>
```

Comment:
>so all non-american english devices get the non-american variant with ou

Is it that big of a deal if we only use `neighboring` to unify this once and for all? Furthermore, did it even make sense renaming and replacing the string naames everywhere then? Don't get me wrong, I know the reasoning you have behind this, I just don't want developers to get confused...